openid-caep-specification-1_0.md

```diff
@@ -758,12 +758,67 @@ The following is a non-normative example of the `session-established` event type
           "ip": "192.168.1.12",
           "fp_ua": "abb0b6e7da81a42233f8f2b1a8ddb1b9a4c81611",
           "acr": "AAL2",
-          "amr": "otp"
+          "amr": "otp",
+          "event_timestamp": 1615304991643
         }
     }
 }
 ~~~
 
+## Session Presented {#session-presented}
+Event Type URI:
+
+`https://schemas.openid.net/secevent/caep/event-type/session-presented`
+
+The Session Presented event signifies that the Transmitter has observed the session to be present at the Transmitter at the time indicated by the `event_timestamp` field in the Session Presented event. Receivers may use this information for reasons that include:
+
+* Detecting abnormal user activity
+* Establishing an inventory of live sessions belonging to a user
+
+### Event Specific Claims {#session-presented-event-specific-claims}
+The following optional claims MAY be present in a Session Presented event:
+
+ip
+: The array of IP addresses of the user as observed by the Transmitter. The value MUST be in the format of an array of strings, each one of which represents the RFC 4001 {{RFC4001}} string represetation of an IP address. (**NOTE**, this can be different from the one observed by the Receiver for the same user because of network translation)
+
+fp_ua
+: Fingerprint of the user agent computed by the Transmitter. (**NOTE**, this is not to identify the session, but to present some qualities of the session)
+
+ext_id
+: The external session identifier, which may be used to correlate this session with a broader session (e.g., a federated session established using SAML)
+
+risk_score
```

Comment:
This doesn't belong here. If you want to convey a risk score then it should be sent in its own event type. Maybe a RiskScoreChanged event?

Reply:
I agree

Reply:
I think placing a risk_score here makes sense as an optional claim being that the Transmitter can derive relative risk based on the identified characteristics.